src/components/Explorer/Explorer.tsx

```diff
@@ -154,14 +154,11 @@ const Explorer = (props: Props) => {
             );
         }
         if (journeys) {
-            const journeyResolvedWithError = journeys.find((j) => "error" in j);
```

Comment:
Under the proposal for expressing navigation failures inside `JourneyInfo`, I think we'd want to keep this code as-is to properly handle errors that are more truly catastrophic than failed navigations.